health.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"net/http"
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+)
+
+func startRestAPI() {
+	router := mux.NewRouter()
+	router.Methods("GET").Path("/health").HandlerFunc(getHealth)
+	router.Use(errorLogger)
+	go func() {
+		log.Fatal(http.ListenAndServe(":9000", router))
+   }()
+}
+
+func getHealth(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is a pretty useless health check; you're running this in a separate goroutine so it's not even impacted in any particular way if the main bulk of the program hangs in a deadlock or something.
I would at least have a periodic liveness ping from your main loop, where getHealth() checks whether the latest liveness ping was sufficiently recent.

Reply:
fixed